canonical-websites/www.ubuntu.com
# Updated  /kubernetes page with new copy

## Done

- Updated k8s index page with new copy

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/kubernetes](http://0.0.0.0:8001/kubernetes)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc k8s](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#) and [copy doc plans-and-pricing](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#heading=h.j34bveun4fte) and accept all suggestions

## Issue / Card

Fixes #4292 and fixes #4293
